content/en/docs/v3.6/tech-overview/api_guarantees.md

```diff
@@ -23,55 +22,55 @@ This specification enumerates the API guarantees made by etcd.
   * [Revoke]
   * [Keep alive](../api/#keep-alives)
 
-KV API allows for direct reading and manipulation of key value store.
-Watch API allows subscribing to key value store changes.
-Lease API allows assigning a time to live to a key.
+The KV API allows for direct reading and manipulation of key value store.
+
+The Watch API allows subscribing to key value store changes.
+
+The Lease API allows assigning a time to live to a key.
 
-Both KV and Watch APIs allow access to not only the latest versions of keys, but
-also previous versions are accessible within a continuous history window, limited
+Both the KV and Watch APIs allow access to the latest versions of keys, plus all previous versions within a continuous history window limited
```

Comment:
Often times we use short lines (~80 chars or so) to help localization teams to make their updates.

Reply:
Restored shorter line lengths in this file.